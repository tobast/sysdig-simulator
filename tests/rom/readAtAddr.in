8
00000000
00001000
00000100
00001100
10000000
01010000
11000000
00010000
