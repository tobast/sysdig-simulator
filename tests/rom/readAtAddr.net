INPUT a
OUTPUT z
VAR
	a:8, z:8
IN
z = ROM 8 8 a
