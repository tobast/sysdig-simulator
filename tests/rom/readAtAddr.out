10000010
11000010
10100010
11100010
00000100
00001011
00010010
01000010
